clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -919,7 +919,7 @@ protected void leaderExecuteTopicSwitch(
   }
 
   @Override
-  protected void processTopicSwitch(
+  protected boolean processTopicSwitch(
```

Comment:
What does a 'false' mean for this function?  Can we javadoc it?

Reply:
Thanks for pointing out. Just added.